scripts/xtensa-build-zephyr.py

```diff
@@ -323,6 +326,19 @@ def west_init_update():
 	# Do NOT "west update sof"!!
 	execute_command(["west", "update", "zephyr", "hal_xtensa"], timeout=300, cwd=west_top)
 
+def get_sof_version(abs_build_dir):
+	"""[summary] Generate SOF FW file version string major.minor.micro from sof_version.h
+	When builing multiple platforms from the same SOF commit, all platforms share the same version.
```

Comment:
```suggestion
	When building multiple platforms from the same SOF commit, all platforms share the same version.
```

Reply:
Function description has been updated and typo is fixed